### PR TITLE
feat: added the features sub project in the micro front end way

### LINKED ISSUE #267: Create the features module in the micro front end way
**Is your feature request related to a problem? Please describe.**
Create the features module in the micro front end way

### PR DESCRIPTION
**Pull request by J-DK**
# Description

Add the features sub project as a separate micro front end

Fixes #267 

## Type of change

Please check options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by paul58914080** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- projects/ff4j-features/e2e/protractor.conf.js  1
- projects/ff4j-features/karma.conf.js  1
         

Clones added
============
- e2e/protractor.conf.js  1
- projects/ff4j-features/e2e/protractor.conf.js  1
- projects/ff4j-features/karma.conf.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/paul58914080/ff4j-web-console/pullRequest?prid=5250526&bid=17206233)

**Comment by paul58914080** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- projects/ff4j-features/e2e/protractor.conf.js  1
- projects/ff4j-features/karma.conf.js  1
         

Clones added
============
- projects/ff4j-features/e2e/protractor.conf.js  1
- projects/ff4j-features/karma.conf.js  2
- e2e/protractor.conf.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/paul58914080/ff4j-web-console/pullRequest?prid=5250526&bid=17206233)

File: package.json [hunk 1]
```diff
@@ -58,6 +62,7 @@
     "@types/node": "^13.9.4",
     "angular-cli-ghpages": "^0.6.2",
     "codelyzer": "^5.1.2",
+    "del": "^5.1.0",
```
[the remark this answers is not in the document]
Comment (J-DK) [reply package.json]: Oh yes Paul, somehow they are missed. I saw that you added in the next commit. Thanks

**Comment by paul58914080** [comment]
Thank you @J-DK for your contribution 🥇

**Pull request merged**